test-helpers/a11y-helper.js

```diff
@@ -0,0 +1,39 @@
+/*eslint no-console: 0*/
+/*eslint no-undef: 0*/
+import 'axe-core/axe.min.js';
+
+/*
+* Rules can be disabled using the options parameter:
+* options = { "rules":
+*   {
+*     "color-contrast": { enabled: false },
+*     "valid-lang": { enabled: false }
+*   }
+* }
+* Rule ids can be found here: https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md
+*/
+export async function runAxe(element, options) {
+	const results = await axe.run(element, options);
+
+	const violations = results.violations;
+
+	console.log('Inapplicable Tests:', results.inapplicable.length);
```

Comment:
No changed needed, but one way to avoid the console statement (and corresponding linting exception) is to use `process.stdout.write(...)`.

Reply:
Oh sorry - this is being imported into browser.